README.md

```diff
@@ -1,73 +1,77 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
```

Comment:
- [ ] **Required:** Kindly replace the placeholder `Project Name` with the title of the project.